AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -171,6 +171,7 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
 
         if let attendeesWithJoinedStatus = newAttendeeMap[AttendeeStatus.joined] {
             let attendeesJoined = attendeesWithJoinedStatus.subtracting(currentAttendeeSet)
+            logger.info(msg: "attendeesJoined: \(attendeesJoined)")
```

Comment:
It's been a hassle to get useful information from ADB logs without this information being logged on Android. So, I decided to add these logs for iOS for better debugging.

Reply:
I don't think they are PII, but I think it might make builder's log more noisy with unnecessary information.

Reply:
If not a PII concern, then it should be fine as it will only be printed once per attendee join/left etc and helps debugging issues from logs. We're not logging volume/signal updates as those can get very noisy.

Let me know if you still think it's better to remove these logs.

Reply:
Just an FYI - Chime Clients treat `attendeeId` as PII. Clients have a framework in place that masks all PII info when being printed to logs. That makes me think we should probably remove these logs from Android as well as iOS SDKs if we can't mask the information while printing. 

Thoughts?